### ISSUE TITLE
padding parameter of conv2d layer is not saved correctly with dilation > 1

### ISSUE THREAD

**Issue by guoyejun** [tensorflow/tensorflow#30491]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):Linux Ubuntu 16.04
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 1.13.1
- Python version: 3.5.2
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

**Describe the current behavior**
tf.layers.conv2d(padding='same') is saved as padding is VALID.

**Describe the expected behavior**
tf.layers.conv2d(padding='same') is saved as padding is SAME.

**Code to reproduce the issue**
i use the below code to generate .pb file, the key is that padding parameter is same, and dilation_rate is 2.
```
import tensorflow as tf
import numpy as np

x = tf.placeholder(tf.float32, shape=[1, None, None, 1])
conv = tf.layers.conv2d(x, 64, 5, activation=tf.nn.tanh, padding='same', dilation_rate=(2, 2), kernel_initializer=tf.keras.initializers.he_normal())
y = tf.identity(conv, name='y')


with tf.Session() as sess:
    sess.run(tf.global_variables_initializer())
    output_graph_def = tf.graph_util.convert_variables_to_constants(sess, sess.graph_def, ['y'])
    tf.train.write_graph(output_graph_def, '.', 'same.pb', as_text=False)
```

I use the following code to load and check the .pb file, but the padding parameter is not same, but VALID.
```
import tensorflow as tf
import numpy as np

with open('same.pb', 'rb') as f:
    graph_def = tf.GraphDef()
    graph_def.ParseFromString(f.read())
    nodes = graph_def.node
    for node in nodes:
        if node.op == 'Conv2D':
            print(node.attr['padding'].s)
```

**Comment by ravikyram** [comment]
I have tried on colab with TF version 1.13.1 and was able to reproduce the issue.Thanks!

**Comment by tensorflowbutler** [comment]
Hi There,

 We are checking to see if you still need help on this, as you are using an older version of tensorflow which is officially considered end of life . We recommend that you upgrade to the latest 2.x version and let us know if the issue still persists in newer versions. Please open a new issue for any help you need against 2.x, and we will get you the right help. 

 This issue will be closed automatically 7 days from now. If you still need help with this issue, please provide us with more information.

**Comment by guoyejun** [comment]
thanks, i'll try 2.x later.
